active/radiobuttons/RadioButtonsAPI.md

```diff
@@ -0,0 +1,182 @@
+# Background
+
+The RadioButtons control is our answer to enabling better keyboarding and narrator support for
+RadioButton elements that need to be presented in a grouped layout.
+
+Currently, the only way to group single RadioButton elements today is through using a StackPanel
+and individually labeling each RadioButton through its GroupName property in order for narrator to
+read out the options somewhat understandably.
+
+However, using that method still leaves gaps in the narrator experience and StackPanel does not
+behave the way users expect when navigating the list via keyboard - RadioButtons was introduced to
+help solve those issues.
+
+# Description
+
+RadioButtons is a new control that enables you to create groups of RadioButton elements easily,
+while also correctly supporting keyboarding and narrator functionality.
+
+# Examples
+
+The following example demonstrates how to create a small collective group of RadioButton elements
+using the RadioButtons control.
+
+```xaml
+<muxc:RadioButtons Header="App Mode" SelectedIndex="2">
+    <x:String>Dark</x:String>
+    <x:String>Light</x:String>
+    <x:String>Windows Default</x:String>         
+</muxc:RadioButtons>
+```
+
+![alt text](DefaultRadioButtonGroup.png)
```

Comment:
Can you update the image to show Dark/Light/Windows Default, instead of Item1/Item2/Items3? And actually show the selected index as selected.